### PR TITLE
Kalyani

### PR DESCRIPTION
**Pull request by kadarikalyani**
would please check my code lawyers site

### CODE REVIEW

**Comment by abhinaykrupa** [comment]
@kadarikalyani there are no file changes ... 
seems like you dint commit the changes from your local ... or may be you forgot to push changes from your kalyani local branch to origin/kalyani

**Comment by kadarikalyani** [comment]
adding lawyer search forms

File: application/models/LawyersModel.php [hunk 1]
```diff
@@ -315,167 +263,56 @@ public function getWebsites($params)
...
-                        .($params['yearMax'] ? $params['yearMax'] : $this->yearCurrent);
-                }
-                if ($params['priceMin'] || $params['priceMax']) {
-                    $this->query[$website_id]['pricerange'] = ($params['priceMin'] ? $params['priceMin'] : '0').'-'
-                        .($params['priceMax'] ? $params['priceMax'] : 'max');
-                }
-                $this->query[$website_id]['zipcode']  = $params['zip'] ? $params['zip'] : $this->zip;
-                $this->query[$website_id]['distance'] = ($params['distance'] && $params['distance'] != 'all') ? $params['distance'] : '';
+                $this->query[$website_id]['query'] = ''; // $this->websiteSearchTypes[$website_id][$params['searchType']];
+//                $this->query[$website_id]['location'] = $params['zip'] ? $params['zip'] : $this->zip;
```
Comment (abhinaykrupa) [inline application/models/LawyersModel.php]: @kadarikalyani code to pick up the zipcode and aop for this site !

File: application/models/LawyersModel.php [hunk 2]
```diff
@@ -315,167 +263,56 @@ public function getWebsites($params)
...
-                $this->uri[$website_id]['exceptions'][]     = 'distance';
-                $this->uri[$website_id]['trailingslash']    = false;
+//                if ($params['zip']) {
+//                    $this->uri[$website_id]['uri2'] .= ' --'.$params['zip'];
+//                }
+//                $this->uri[$website_id]['searchtype']       = $this->websiteSearchTypes[$website_id][$params['searchType']];
+//                $this->uri[$website_id]['ucwords']          = true;
+//                $this->uri[$website_id]['replacespacewith'] = '-';
+//                $this->uri[$website_id]['exceptions'][]     = 'uri1';
+//                $this->uri[$website_id]['trailingslash']    = false;
```
Comment (abhinaykrupa) [inline application/models/LawyersModel.php]: @kadarikalyani remove all this unnecessary code if you dont need this

**Comment by abhinaykrupa** [comment]
@kadarikalyani review comments and make appropriate changes

**Comment by kadarikalyani** [comment]
removed all whatever you comment the changes

**Pull request merged**